lacommunaute/surveys/models.py

```diff
@@ -59,6 +61,11 @@ class DSP(DatedModel):
         choices=enums.DSPAvailability.choices,
         verbose_name="disponibilité",
     )
+    location = models.CharField(max_length=255, verbose_name="localisation")
+    city_code = models.CharField(max_length=5, verbose_name="city code", null=True, blank=True)
```

Comment:
Le `verbose_name="city code"` est ce que django utilisera par défaut.
```suggestion
    city_code = models.CharField(max_length=5, null=True, blank=True)
```